base/lttagging.dtx

```diff
@@ -104,49 +104,55 @@
 %  debugging strings\ldots
 %
 %
+% \DescribeMacro\UseExpandableTaggingSocket
 % \DescribeMacro\UseTaggingSocket
+% \DescribeMacro\tag_socket_use_expandable:n
 % \DescribeMacro\tag_socket_use:n
 % \DescribeMacro\tag_socket_use:nn
-%  To support tagging in packages we use sockets with names starting
-%  with \texttt{tagsupport/}. Usually, these sockets have exactly two
-%  plugs defined:
-%  \plug{noop} (when no tagging is requested or tagging is not wanted
-%  for some reason) and a second plug that enables the tagging. There
-%  may be more, e.g., tagging with special debugging, etc., but right
-%  now it is usually just on or off.
-%
-%  Given that we sometimes have to suspend tagging, it would be fairly
-%  inefficient to put different plugs into these sockets whenever that
-%  happens. We therefore offer \cs{UseTaggingSocket} which is like
-%  \cs{UseSocket} except that the socket name is specified without
-%  \texttt{tagsupport/}, i.e.,
+% \DescribeMacro\tag_socket_use:nnn
+% Given that we sometimes have to suspend tagging, it would be fairly
+% inefficient to put different plugs into these sockets whenever that
+% happens. We therefore offer \cs{UseTaggingSocket} which is like
+% \cs{UseSocket} except that is expects a socket starting with
+% \texttt{tagsupport/} but the socket name is specified without
+% this prefix, i.e.,
 % \begin{quote}
 %   \verb=\UseTaggingSocket{foo}=      $\to$
 %   \verb=\UseSocket{tagsupport/foo}=
-% \end{quote}
-%  Beside being slightly shorter, the big advantage is that this way
-%  we can change \cs{UseTaggingSocket} to do nothing when tagging is
-%  suspended with \cs{SuspendTagging} instead of changing the plugs of
-%  the tagging support sockets back and forth.
+% \end{quote}.
 %
-%  This means that with \cs{SuspendTagging} in force all socket
-%  arguments are ignored (including the case when there is a single
-%  one), i.e., all sockets used with \cs{UseTaggingSocket} behave as
-%  if the \text{noop} plug is assigned.
+%  Beside being slightly shorter, the big advantage is that this way
+%  we can change \cs{UseTaggingSocket} to do nothing by switching a boolean
+%  instead of changing the plugs of the tagging support sockets back and forth.
+%
+% Usually, these sockets have exactly two plugs defined.
+% When using \cs{UseTaggingSocket} the first plug is always
+% used if tagging is suspended.
```

Comment:
That description doesn't really make sense to me. The point of \SuspendTagging is that no plugs are changed but the the behavior of \UseTaggingSocket is changed. It is not a defined "plug" that is used but rather that no plug is used but that the socket stops acting like a socket more or less. While one can describe the behavior as if there is a plug but in practice the plug may not exist and on it only behaves as if the plug is assigned (which is what I wrote in my version of the description).

Reply:
I agree that it is misleading. I changed the wording in the tagpdf documentation ([tagpdf-user, currently in the taggingsocket branch](https://github.com/latex3/tagpdf/commit/6522fabb82092e98b3b2073458f11e62f1a07347)). If that sounds ok, it can be copied here.

Reply:
I made one suggestion for change in your text @u-fischer, otherwise I think that's fine

Reply:
Adjusted following Ulrike's `tagpdf` change.